examples/vitepress/.vitepress/config.js

```diff
@@ -0,0 +1,65 @@
+require('esbuild-register')
+
+const Entry1 = [
```

Comment:
Well I feel the example is a bit too verbose, giving we only provided only a few API, maybe we don't need this example?